modules/widgets/wordpress-post-widget.php

```diff
@@ -27,138 +73,600 @@ public function __construct() {
 				'description' => __( 'Displays a list of recent posts from another WordPress.com or Jetpack-enabled blog.', 'jetpack' ),
 			)
 		);
+
+		/**
+		 * Check the status of the cron task.
+		 */
+		self::check_for_cron();
```

Comment:
Updated in db947f10d69fbc38dbaf502a9fcdd9384ddbe55a. 

I moved the check to happen on `wp_loaded` action, but I am still not sure this is the best way to handle it.